akka-projection-cassandra/src/main/scala/akka/projection/cassandra/scaladsl/CassandraProjection.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+package akka.projection.cassandra.scaladsl
+
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+import akka.Done
+import akka.annotation.ApiMayChange
+import akka.projection.Projection
+import akka.projection.ProjectionId
+import akka.projection.cassandra.internal.CassandraProjectionImpl
+import akka.stream.scaladsl.Source
+
+@ApiMayChange
+object CassandraProjection {
+
+  def atLeastOnce[Offset, StreamElement](
+      projectionId: ProjectionId,
+      sourceProvider: Option[Offset] => Source[StreamElement, _],
+      offsetExtractor: StreamElement => Offset,
+      saveOffsetAfterNumberOfEvents: Int,
+      saveOffsetAfterDuration: FiniteDuration)(handler: StreamElement => Future[Done]): Projection[StreamElement] =
```

Comment:
We could have something similar to `CommitterSettings` from Alpakka Kafka. Just for the sake of keeping API similarity. Something to think and explore on another PR eventually.

Reply:
:+1: 

These APIs are collecting a lot of parameters, but it's fine for now.

Reply:
created issue https://github.com/akka/akka-projection/issues/59 for future consideration